tests/test_sanity.py

```diff
@@ -24,7 +23,7 @@
 from tools import response_file
 
 SANITY_FILE = shared.Cache.get_path('sanity.txt')
-commands = [[EMCC], [path_from_root('tests/runner'), 'blahblah']]
+commands = [[EMCC], [path_from_root('tests', 'runner'), 'blahblah']]
```

Comment:
Did you mean to do revert that change to use "/" in pathsnames?   Perhaps a bad merge?